mmv1/third_party/terraform/services/migrationcenter/resource_migration_center_group_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package migrationcenter_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccMigrationCenterGroup_migrationGroupUpdate(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckMigrationCenterGroupDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMigrationCenterGroup_migrationGroupBasicExample(context),
+			},
+			{
+				ResourceName:            "google_migration_center_group.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"location", "group_id", "labels", "terraform_labels"},
```

Comment:
I might be missing something, but why are we ignoring `location` and `group_id`?

Reply:
You know, I'm not actually sure. When I create these tests I generally generate the generated_test file and then copy that over for the basic structure, and that file includes these ignores: https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-9425-old..auto-pr-9425#diff-88e8f972e0ab2bf8e14ceeafa49d9d8f4e2fd32235477adfbe6e6e04f93a6291R52

I'm not sure why those are there though, maybe because they're marked as url_param_only?

Reply:
Ah you're right, it looks like all `url_param_only` fields end up here in the generated tests, because they are not read from the API. I suspect that they don't need to be ignored in this case because they are part of the import id, but I suppose we don't always know that, and this strategy is safer.

Seems fine to leave.